app/javascript/src/ActiveDocs/OAS3Autocomplete.ts

```diff
@@ -120,3 +120,15 @@ export const autocompleteOAS3 = async (response: SwaggerUIResponse, accountDataU
     text: JSON.stringify(body)
   }
 }
+export const autocompleteInterceptor = async (response: SwaggerUIResponse, specUrl: string | undefined, accountDataUrl: string, serviceEndpoint: string): Promise<Response> => {
```

Comment:
```suggestion
export const autocompleteInterceptor = async (response: SwaggerUIResponse, specUrl?: string, accountDataUrl: string, serviceEndpoint: string): Promise<Response> => {
```

Reply:
This gives a warning `A required parameter cannot follow an optional parameter`.

But I'll just set a default `''` value to `url`, and it will be OK.
This `url` shouldn't be empty, but if for some reason it is, a nice error `No API definition provided.` is shown on the page.